### DISCUSSION TITLE
git safe.directory for grammars

### DISCUSSION THREAD

**Discussion by N4tus** [helix-editor/helix#3039] [Q&A]
cloning the repo and then running `cargo install --path helix-term` fails with git complaining that `/d/dev/rust/helix/runtime/grammars/sources/c` is owned by someone else and I should run `git config --global --add safe.directory /d/dev/rust/helix/runtime/grammars/sources/c` to add this repo as an exception. I did this and now the next grammar also needs this, as well as each grammar helix want to fetch. Doing the manually is tedious but a search on how to use wild-cards or something similar for the git command yielded no result. 

What is the best way around that issue?

**Comment by the-mikedavis** [comment]
You can disable the check altogether with

```
git config --global --add safe.directory '*'
```

See also https://stackoverflow.com/questions/71855882/how-to-add-directory-recursively-on-git-safe-directory

As far as I know, globbing is not supported though.
